bitnami/zookeeper/values.yaml

```diff
@@ -398,6 +398,7 @@ livenessProbe:
   failureThreshold: 6
   successThreshold: 1
   probeCommandTimeout: 2
+  useAdmin: false
```

Comment:
I'm indifferent to the value name 'useAdmin' and am happy to change as needed.

While I did add two separate 'useAdmin' toggles in this commit, it was to try to match the existing convention and I remain convinced that a single toggle would be safer (so liveness and readiness are measured in the same fashion).

Reply:
I'd rather keep two separate parameters. It gives the user more freedom to customize the probes.